cpp/src/io/orc/reader_impl.cu

```diff
@@ -725,12 +725,16 @@ struct list_buffer_data {
 void generate_offsets_for_list(device_span<list_buffer_data const> buff_data,
```

Comment:
The call site of `generate_offsets_for_list` copies buffer data from host to device. You can just skip the round trip and pass a host span instead.

Reply:
Great call, I missed that.

Reply:
Done.